aws/resource_aws_cognito_user_group_test.go

```diff
@@ -215,7 +215,7 @@ resource "aws_iam_role" "group_role" {
       "Action": "sts:AssumeRoleWithWebIdentity",
       "Condition": {
         "StringEquals": {
-          "cognito-identity.amazonaws.com:aud": "us-east-1:12345678-dead-beef-cafe-123456790ab"
```

Comment:
Nit: Generally preferable to use the `aws_region` data source instead of passing `testAccGetRegion()` through the configuration function.

Reply:
Absolutely! 

With data source:
```
--- PASS: TestAccAWSCognitoUserGroup_complex (34.02s)
```